pkg/manifest/manifest.go

```diff
@@ -98,6 +110,69 @@ func (m *Manifest) UnmarshalJSON(in []byte) error {
 	return validateResourceId(m.id)
 }
 
+func (m *Manifest) Include(excludeIdentifier *string, includeTechPreview bool, profile *string, capabilities *ClusterCapabilities) error {
+
+	annotations := m.Obj.GetAnnotations()
+	if annotations == nil {
+		return fmt.Errorf("no annotations")
+	}
+
+	excludeAnnotation := fmt.Sprintf("exclude.release.openshift.io/%s", *excludeIdentifier)
```

Comment:
you need to wrap all this in an `if excludeIdentifier != nil { ... }` block to avoid panicking if `excludeIdentifier` is nil.  Same for the other pointer arguments.

Reply:
fixed in https://github.com/openshift/library-go/commit/f529590c5da24451b978dd972aade0a4bc3d40bd